modules/microshift-stopping-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-stopping-lvms-csi-snapshot_{context}"]
+== Stopping deployments that run CSI implementations
+
+User workloads with attached volumes must be manually stopped before disabling the CSI driver or CSI snapshot implementation pods. Use the following procedure to disable installation of one or both CSI components.
+
+[NOTE]
+====
+MicroShift will not delete CSI implementation pods. MicroShift must be configured to disable installation of one or both of the CSI pods during the start-up process.
+====
+
+.Procedure
+
+. To disable installation of the CSI driver, enter the following `driver` value under the `storage` section of the {microshift-short} configuration file:
++
+[source,yaml]
+----
+# ...
+  storage
+   driver:'' <1>
+# ...
+----
+<1> Valid values are `none` or `lvms`.
++
+[NOTE]
+====
+If the `driver` value is empty or null, {microshift-short} defaults to deploying LVMS.
+====
+
+. To disable installation of the CSI snapshot controller, enter the following `optionalCsiComponents` value under the `storage` section of the {microshift-short} configuration file:
++
+[source,yaml]
+----
+# ...
+  storage
+   optionalCsiComponents:'' <1>
+# ...
+----
+<1> Accepted values are `csi-snapshot-controller`, `csi-snapshot-webhook`, or `none`. `none` is mutually exclusive with all other values.
++
+[NOTE]
+====
+If the `optionalCsiComponents` value is empty or null, {microshift-short} defaults to deploying snapshot-controller and snapshot-webhook.
+====
+
+. To restart {microshift-short} after the `driver` and `optionalCsiComponents` fields are specified with supported values in the {microshift-short} config file, run the following command:
```

Comment:
```suggestion
. To restart {microshift-short} after the `driver` and `optionalCsiComponents` fields are specified with supported values in the {microshift-short} configuration file, run the following command:
```

Not sure how "config" holds up with translation to other languages.

Reply:
done